src/css/object/utility/utility.scss

```
@@ -1,2 +1,12 @@
 @forward '../../foundation/_variables.scss';
-@forward '../../foundation/_mixins.scss';
\ No newline at end of file
+@forward '../../foundation/_mixins.scss';
+
+.u-wrapper {
+  width: 80%;
+  margin: auto;
+}
+
+a:hover {
+  opacity: 0.8;
+  transition: 1s;
```

[must]

仕様上、`transition: .3s` なので、他の箇所も合わせて修正をお願いします

```suggestion
  transition: .3s;
```

また、ホバーが外れた時に、transition が効いていない箇所がありそうなので、ホバーイベントがある要素を全部見直してもらって修正をお願いします！

https://github.com/gizumo-education/front-coding/assets/48352529/2bb09f1c-8cfa-46f3-a379-8f80c4365e83